lib/connection.js

```diff
@@ -77,7 +77,9 @@ Connection.prototype.callMethod = function(
 ) {
   const packet = this.createPacket('call', interfaceName, methodName, args);
   const packetId = packet.call[0];
-  this._callbacks[packetId] = callback || common.doNothing;
+  this._callbacks[packetId] = callback || ((error) => {
+    this.emit('error', error);
```

Comment:
With this implementation it looks like that whenever someone doesn't provide callback, it always emits error on callback call. Maybe you should check if `error` is not `undefined` or `null` and emit it only in that case.
Or did I get this wrong?